README.md

```diff
@@ -565,7 +567,9 @@ appHistory.addEventListener("currententrychange", e => {
 });
 ```
 
-_TODO: could we populate this for cross-document navigations too? Then it kind of overlaps with existing timing APIs, and is probably a lot harder to implement..._
+_TODO: reconsider cross-document navigations. There will only be one (the initial load of the page); should we even fire this event in that case? (That's [#31](https://github.com/WICG/app-history/issues/31).) Could we give `startTime` a useful value there, if we do?_
+
+_TODO: is this property-on-the-event design the right one, or should we integrate with the performance timeline APIs, or...? Discuss in [#33](https://github.com/WICG/app-history/issues/33)._
```

Comment:
We should totally integrate this with the performance APIs, but that doesn't necessarily mean this timestamp is not needed. Are there other use cases for this timestamp, other than reporting? 
e.g. I know that we have [`Event.timestamp`](https://developer.mozilla.org/en-US/docs/Web/API/Event/timeStamp), which is awfully similar.